labelbox/client.py

```diff
@@ -1786,3 +1789,104 @@ def get_batch(self, project_id: str, batch_id: str) -> Entity.Batch:
             experimental=True)["project"]["batches"]["nodes"][0]
 
         return Entity.Batch(self, project_id, batch)
+
+    def send_to_annotate_from_catalog(
+            self,
+            batch_name: str,
+            data_rows: DataRowIdentifiers,
+            source_model_run_id: Optional[str],
+            source_project_id: Optional[str],
+            destination_project: str,
+            task_queue_id: Optional[str],
+            exclude_data_rows_in_project: bool = False,
+            ontology_mapping: Dict[str, str] = None,
+            override_existing_annotations_rule:
+        ConflictResolutionStrategy = ConflictResolutionStrategy.KeepExisting,
+            priority: int = 5):
+
+        mutation_str = """mutation SendToAnnotateFromCatalogPyApi($input: SendToAnnotateFromCatalogInput!) {
+                            sendToAnnotateFromCatalog(input: $input) {
+                              taskId
+                            }
+                          }
+        """
+
+        destination_task_queue = {
+            "type": "id",
+            "value": task_queue_id
+        } if task_queue_id else {
+            "type": "done"
+        }
+
+        annotations_input = {
+            "projectId":
+                source_project_id,
+            "featureSchemaIdsMapping":
+                ontology_mapping if ontology_mapping else {},
+        } if source_project_id else None
+
+        predictions_input = {
+            "featureSchemaIdsMapping":
+                ontology_mapping if ontology_mapping else {},
+            "modelRunId":
+                source_model_run_id,
+            "minConfidence":
+                0,
+            "maxConfidence":
+                1
+        } if source_model_run_id else None
+
+        if isinstance(data_rows, UniqueIds):
```

Comment:
I believe I've come across this code snippet in a previous PR for Model Foundry. Can we create a method and reuse it in both places?